include/ppx/grfx/grfx_command.h

```diff
@@ -191,6 +191,13 @@ struct RenderingInfo
     grfx::DepthStencilClearValue DSVClearValue                          = {1.0f, 0xFF};
 };
 
+struct DynamicRenderPassInfo
+{
+    grfx::Rect                             mRenderArea        = {};
+    std::vector<grfx::RenderTargetViewPtr> mRenderTargetViews = {};
+    grfx::DepthStencilViewPtr              mDepthStencilView  = nullptr;
+};
```

Comment:
Could you move this to line 593 to be a nested class. I don't think there is a good reason to expose this in a public interface, and DynamicRenderPassInfo keyword can be used for other things.